Mathlib/SetTheory/Ordinal/Arithmetic.lean

```diff
@@ -461,6 +461,7 @@ alias IsLimit.add := add_isLimit
 
 
 /-- The set in the definition of subtraction is nonempty. -/
+@[deprecated (since := "2024-09-11")]
 theorem sub_nonempty {a b : Ordinal} : { o | a ≤ b + o }.Nonempty :=
```

Comment:
I would make this private rather than deleting it

Reply:
This theorem isn't even used anywhere.